CHANGES.rst

```diff
@@ -13,7 +13,9 @@ New Features
 
 - The ``specviz.load_spectrum`` method is deprecated; use ``specviz.load_data`` instead. [#2273]
 
-- Add first-pass launcher to select config and auto-identify data. [#2257]
+- Add launcher to select config and require --layout argument when launching standalone. [#2257]
+
+- Launcher displays which configs are compatible with a provided dataset. [#2267]
```

Comment:
I think these two entries can be merged.

Reply:
Good idea, done in https://github.com/spacetelescope/jdaviz/pull/2267/commits/942751809f54a6c8c7abb712f9ea295158bd8c39